.github/workflows/pypi_release.yaml

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: MIT
+
```

Comment:
License information is not needed. 

(Btw, DEM is licensed under 'Eclipse Public License - v 2.0')

Reply:
@janosmurai Fixed this with the latest commit.